src_web/kamaho-shokusu/src/Policy/MUserInfoPolicy.php

```
@@ -85,6 +85,11 @@ public function canUpdateSystemAdminStatus(?IdentityInterface $user, MUserInfo $
         return $this->isSystemAdmin($user);
     }
 
+    public function canUpdateReportAccess(?IdentityInterface $user, MUserInfo $resource): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$resource'. (undefined)

(UnusedFormalParameter)